indra/newview/viewer_manifest.py

```diff
@@ -492,6 +503,10 @@ def test_for_no_msvcrt_manifest_and_copy_action(self, src, dst):
         else:
             print("Doesn't exist:", src)
         
+    # This differs for Windows, it delivers XML files as resources
+    def is_packaging_xml_files(self):
+        return False;
```

Comment:
First thing to set to return True after merging into Firestorm...

Reply:
Can you specify reasons? Benefits are a bit questionable given that everything else stays as separate files, but I don't think there are any disadvantages either as long as preview and localization tools keep working. I feel like moving skin files into resources can be a potential issue since people will have to get repo's files now, but I don't know that for sure, agruably you should be working with viewer's repo if you are translating.

Reply:
Can you specify how this should fix issues with the installer or the users anti virus tool borking installations, given that DLLs might also not be installed properly and preventing the viewer from starting. Or SLPLugin.exe regularly getting flagged as malicious and deleted/moved to a quarantine folder?

This just seems to prevent from users customizing the viewer, e.g. dropping in custom skins they created, or simply update translations which will be done by LL either once in a blue moon, might not fit or are simply wrong. I will for sure undo this.

Reply:
> Can you specify how this should fix issues with the installer

That was my argument as well. It does make viewer slightly more reliable (ex: against things that would modify text files and warranties that warning strings load), but most of the time it doesn't fix anything and just bumps problem to the next file (.msg in this case) instead of fixing our install process and validation.

> This just seems to prevent from users customizing the viewer

Does it actually happen? This code is supposed (haven't checked it yet) to load local files before resources so it should still be possible to have custom files. To me this change does feel like a disadvantage in terms of customizability, but I don't know if anyone actually modifies those files and won't fail a review on a feeling.

Reply:
> > This just seems to prevent from users customizing the viewer
> 
> Does it actually happen? This code is supposed (haven't checked it yet) to load local files before resources so it should still be possible to have custom files. To me this change does feel like a disadvantage in terms of customizability, but I don't know if anyone actually modifies those files and won't fail a review on a feeling.

Yes. Or change shortcuts in the menu because they don't like them.  With that change you would have to checkout or download a zipped version of the repository and make sure you grab the files for the correct version seems a bit... cumbersome.

Reply:
This fix is specific to windows. Is the problem specific to windows?

Reply:
I'm skeptical of the need to handle this case rather than just warning about it. How hard would it be to just detect the missing file case and tell the user to reinstall?

Reply:
> How hard would it be to just detect the missing file case and tell the user to reinstall?

That's what I did in DRTVWR-591, [here](https://github.com/secondlife/viewer/commit/51088cde7f5a0bdaf9249bfdd5d31b9b212403ab)

> This fix is specific to windows. Is the problem specific to windows?

Mostly, it's overwhelmingly more frequent on windows with couple cases on MacOS.

Reply:
@Ansariel , for those users who want to experiment with XML files without checking out the repository I would like to propose the feature "Export XML files", some button which will unpack all XML contents from resources to external files in proper directories

Reply:
As you can see, if there is a file in the directory and the correspondent resource, the file has a priority, and it will be used, not the resource
This is done specially for developers working with XML files at run-time